web/src/utils/tcp-drop.ts

```diff
@@ -0,0 +1,224 @@
+  },
+  { value: 54, name: 'SKB_DROP_REASON_DEV_HDR', description: 'device driver specific header/metadata is invalid' },
+  {
+    value: 55,
+    name: 'SKB_DROP_REASON_DEV_READY',
+    description:
+      // eslint-disable-next-line max-len
+      'the device is not ready to xmit/recv due to any of its data structure that is not up/ready/initialized, e.g., the IFF_UP is not set, or driver specific tun->tfiles[txq] is not initialized'
+  },
+  { value: 56, name: 'SKB_DROP_REASON_FULL_RING', description: 'ring buffer is full' },
+  { value: 57, name: 'SKB_DROP_REASON_NOMEM', description: 'error due to OOM' },
+  {
+    value: 58,
+    name: 'SKB_DROP_REASON_HDR_TRUNC',
+    description:
+      'failed to trunc/extract the header from networking data, e.g., failed to pull the protocol header from frags via pskb_may_pull()'
+  },
+  {
+    value: 59,
+    name: 'SKB_DROP_REASON_TAP_FILTER',
+    description: 'dropped by (ebpf) filter directly attached to tun/tap, e.g., via TUNSETFILTEREBPF'
+  },
+  {
+    value: 60,
+    name: 'SKB_DROP_REASON_TAP_TXFILTER',
+    description: 'dropped by tx filter implemented at tun/tap, e.g., check_filter()'
+  },
+  { value: 61, name: 'SKB_DROP_REASON_ICMP_CSUM', description: 'ICMP checksum error' },
+  {
+    value: 62,
+    name: 'SKB_DROP_REASON_INVALID_PROTO',
+    description: "the packet doesn't follow RFC 2211, such as a broadcasts ICMP_TIMESTAMP"
+  },
+  {
+    value: 63,
+    name: 'SKB_DROP_REASON_IP_INADDRERRORS',
+    description: 'host unreachable, corresponding to IPSTATS_MIB_INADDRERRORS'
+  },
+  {
+    value: 64,
+    name: 'SKB_DROP_REASON_IP_INNOROUTES',
+    description: 'network unreachable, corresponding to IPSTATS_MIB_INADDRERRORS'
+  },
+  { value: 65, name: 'SKB_DROP_REASON_PKT_TOO_BIG', description: 'packet size is too big (maybe exceed the MTU)' },
+  { value: 66, name: 'SKB_DROP_REASON_DUP_FRAG', description: 'duplicate fragment' },
+  { value: 67, name: 'SKB_DROP_REASON_FRAG_REASM_TIMEOUT', description: 'fragment reassembly timeout' },
+  {
+    value: 68,
+    name: 'SKB_DROP_REASON_FRAG_TOO_FAR',
+    description: 'ipv4 fragment too far. (/proc/sys/net/ipv4/ipfrag_max_dist)'
+  },
+  {
+    value: 69,
+    name: 'SKB_DROP_REASON_TCP_MINTTL',
+    description: 'ipv4 ttl or ipv6 hoplimit below the threshold (IP_MINTTL or IPV6_MINHOPCOUNT)'
+  },
+  { value: 70, name: 'SKB_DROP_REASON_IPV6_BAD_EXTHDR', description: 'Bad IPv6 extension header' },
+  { value: 71, name: 'SKB_DROP_REASON_IPV6_NDISC_FRAG', description: 'invalid frag (suppress_frag_ndisc)' },
+  { value: 72, name: 'SKB_DROP_REASON_IPV6_NDISC_HOP_LIMIT', description: 'invalid hop limit' },
+  { value: 73, name: 'SKB_DROP_REASON_IPV6_NDISC_BAD_CODE', description: 'invalid NDISC icmp6 code' },
+  { value: 74, name: 'SKB_DROP_REASON_IPV6_NDISC_BAD_OPTIONS', description: 'invalid NDISC options' },
+  { value: 75, name: 'SKB_DROP_REASON_IPV6_NDISC_NS_OTHERHOST', description: 'NEIGHBOUR SOLICITATION' }
```

Comment:
do we have to maintain this translation from number to string in two components ? if it makes more sense to have it here we probably removed from FLP and just keep it as a number to reduce future efforts adding new drops ?

Reply:
That's a tradeoff to decide. If we want the user to be able to type either `number` or `name` in the filters, we need these in console plugin.

Reply:
I think we need to be able to filter on both as u have